android/src/main/java/com/appgoalz/rnjwplayer/RNJWPlayerModule.java

```diff
@@ -145,6 +145,10 @@ public void execute (NativeViewHierarchyManager nvhm) {
             playerView.mPlayer.stop();
             playerView.resumeOnFocusGain = false;
           }
+
+          if (playerView.mPlayer.getFullscreen()) {
+            playerView.mPlayer.setFullscreen(false, false);
```

Comment:
Clovece to to sa mi nezda... To to ma podla mna riesit tato cast kodu..

    @Override
    public boolean dispatchKeyEvent(KeyEvent event) {
        // Exit fullscreen or perform the action requested
        if (event.getKeyCode() == KeyEvent.KEYCODE_BACK && this.getFullscreen()) {
            if (event.getAction() == KeyEvent.ACTION_UP) {
                this.setFullscreen(false,false);
                return false;
            }
            return true;
        }
        return super.dispatchKeyEvent(event);
    }